services/backend/Dockerfile

```diff
@@ -18,5 +18,5 @@ COPY pyproject.toml .
 
 COPY src/ .
 
-COPY wait_for_postgres.sh /
-RUN chmod +x /wait_for_postgres.sh
+# COPY wait_for_postgres.sh /
```

Comment:
Good work. This will fix the problem with the inability for our backend docker image to be built. 

Note, we can just delete these lines as they are not needed